core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchReaderIterator.java

```diff
@@ -248,6 +251,10 @@ private void binRanges(TabletLocator tabletLocator, List<Range> ranges,
 
     int lastFailureSize = Integer.MAX_VALUE;
 
+    Retry retry = Retry.builder().infiniteRetries().retryAfter(100, MILLISECONDS)
+        .incrementBy(100, MILLISECONDS).maxWait(1, SECONDS).backOffFactor(1.07)
```

Comment:
I think the max wait should be higher.  If lots are batch scan client are repeatedly failing, it would be good if their collective pressure is lower.

```suggestion
        .incrementBy(100, MILLISECONDS).maxWait(10, SECONDS).backOffFactor(1.07)
```

Reply:
Changed in 65b79ae